debug.html

```diff
@@ -0,0 +1,285 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
+    <title>Debug</title>
```

Comment:
Not sure but not like this,  you have to write this `Debug | Real Dev Squad` or `Real Dev Squad | Debug`. confirm with anyone member